rootfs.go

```diff
@@ -62,16 +62,17 @@ func doRootfs(device installer.UInstaller, args runOptionsType, dt string) error
 		log.Infof("Start updating from local image file: [%s]", updateLocation)
 		image, imageSize, err = FetchUpdateFromFile(updateLocation)
 
-		log.Debugf("Feting update from file results: [%v], %d, %v", image, imageSize, err)
+		log.Debugf("Fetching update from file results: [%v], %d, %v", image, imageSize, err)
 	}
 
 	if image == nil || err != nil {
 		return errors.Wrapf(err, "rootfs: error while updating image from command line")
 	}
 	defer image.Close()
 
+	fmt.Fprintf(os.Stdout, "Installing update from the artifact of size %d\n", imageSize)
 	p := &utils.ProgressWriter{
-		Out: os.Stderr,
+		Out: os.Stdout,
```

Comment:
Which logs are printed to stderr? I had the impression that those are printed to stdout. This was to unify the behavior.